roles/postgresql/tasks/main.yml

```diff
@@ -17,17 +17,26 @@
 - become: true
   become_user: postgres
   block:
-    - name: Create PostgreSQL database
-      postgresql_db:
-        name: "{{ database_name }}"
-
     - name: Create PostgreSQL users
       postgresql_user:
+        state: present
         name: "{{ database_user }}"
         password: "{{ database_password }}"
-        db: "{{ database_name }}"
         encrypted: yes
-        priv: ALL
+
+    - name: Create PostgreSQL database
+      postgresql_db:
+        state: present
+        name: "{{ database_name }}"
+        owner: "{{ database_user }}"
+
+    - name: Grant {{ database_user }} user access to database {{ database_name }}
+      postgresql_privs:
+        type: database
+        database: "{{ database_name }}"
+        roles: "{{ database_user }}"
+        grant_option: no
+        privs: ALL
```

Comment:
@Senen I've run the tests without these lines, and they pass :thinking:. Are these lines necessary, or is it enough to change the order and create the user and then the database?

Reply:
Yes, I checked the `deploy` permissions for tables, schemas and with and without this task and there is no difference.

Reply:
I've just pushed a version removing the task.